File: src/main/scala/net/machinemuse/powersuits/item/ItemPowerArmorLeggings.scala
package net.machinemuse.powersuits.item

import cpw.mods.fml.relauncher.{Side, SideOnly}
import net.machinemuse.utils.render.MuseRenderer
import net.minecraft.client.renderer.texture.IIconRegister

class ItemPowerArmorLeggings extends ItemPowerArmor(0, 2) {
  val iconpath = MuseRenderer.ICON_PREFIX + "armorlegs"

  setUnlocalizedName("powerArmorLeggings")


  @SideOnly(Side.CLIENT)
  override def registerIcons(iconRegister: IIconRegister) {
    itemIcon = iconRegister.registerIcon(iconpath)
  }
}

File: src/main/java/net/machinemuse/powersuits/item/ItemPowerArmorLeggings.java
package net.machinemuse.powersuits.item;

import cpw.mods.fml.relauncher.Side;
import cpw.mods.fml.relauncher.SideOnly;
import net.machinemuse.utils.render.MuseRenderer;
import net.minecraft.client.renderer.texture.IIconRegister;

/**
 * Ported to Java by lehjr on 10/26/16.
 */
public class ItemPowerArmorLeggings extends ItemPowerArmor {
    final String iconpath = MuseRenderer.ICON_PREFIX + "armorlegs";

    public ItemPowerArmorLeggings() {
        super(0, 2);
        setUnlocalizedName("powerArmorLeggings");
    }

    @SideOnly(Side.CLIENT)
    @Override
    public void registerIcons(IIconRegister iconRegister) {
        itemIcon = iconRegister.registerIcon(iconpath);
    }
}
